docs/toc.md

```diff
@@ -989,14 +989,14 @@
 ##### [in](csharp/language-reference/keywords/in.md)
 
 ### [C# Operators](csharp/language-reference/operators/index.md)
-#### [Operator1](csharp/language-reference/operators/index-operator.md)
+#### [[] Operator](csharp/language-reference/operators/index-operator.md)
```

Comment:
This seems to work for me in docfx, but I'm surprised by that (I would expect some kind of escaping of square brackets to be necessary here). Is it also fine on review.docs.MS?